src/views/catalog/CreateFromInstanceTypes/components/BootableVolumeList/hooks/useBootVolumeSortColumns.ts

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
```

Comment:
this hook can return the sorted data, instead of doing slice, sort in the component